internal/controller/backup/cronjob_test.go

```diff
@@ -37,20 +38,20 @@ func TestGCSBackupCronGsql(t *testing.T) {
 	instance.Spec.Google = &kindav1beta1.GoogleInstance{InstanceName: "google-instance-1"}
 	dbcr.Spec.Instance = "staging"
 	dbcr.Spec.Backup.Cron = "* * * * *"
-
+	log := logr.New(logr.Discard().GetSink())
 	os.Setenv("CONFIG_PATH", "./test/backup_config.yaml")
-	conf := config.LoadConfig()
+	conf, _ := config.LoadConfig(log)
```

Comment:
hmmm, don't know if passing the log instance as parameter is a good way to go.

Reply:
gone